### ISSUE TITLE
Training problems

### ISSUE THREAD

**Issue by HongyiDuanmu26** [AlexeyAB/darknet#1260]
9: 495.896484, 496.732147 avg loss, 0.000000 rate, 6.057290 seconds, 576 images
Loaded: 0.000048 seconds
Region 82 Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.513887, .5R: -nan, .75R: -nan,  count: 0
Region 94 Avg IOU: 0.379267, Class: 0.520499, Obj: 0.547163, No Obj: 0.533058, .5R: 0.166667, .75R: 0.000000,  count: 6

Above is output I got from training. The loss is large and won't go down for several iterations and why is the learning rate so close to zero.
I haven't changed other parameters in cfg file except filter number and classes. It is a two class classification problem. Did I miss something or what can I do to fix it.

**Comment by AlexeyAB** [comment]
Train about 2000 iterations.
Or remove line `burn_in=1000` in your cfg-file.

**Comment by HongyiDuanmu26** [comment]
@AlexeyAB WOW, thank you so much! It works!
Would you mind explaining about what it is? Is it a warm-up mechanism?
Thank you very much for your quick reply!

**Comment by eyalzubia** [comment]
Burn-in lowers the learning rate at the beginning of the training (until step 1000 or what is specified in this property). Instead of the given learning rate (lr), it is lr*(batch_num/burn_in)^4 , so it starts low and increases to the specified lr at the end of the burn in period. 
https://github.com/AlexeyAB/darknet/blob/6682f0b98984e2b92049e985b21ed81b76666565/src/network.c#L94
